cpp/src/barretenberg/honk/proof_system/ultra_verifier.hpp

```diff
@@ -21,9 +21,9 @@ template <typename Flavor> class UltraVerifier_ {
     bool verify_proof(const plonk::proof& proof);
 
     std::shared_ptr<VerificationKey> key;
-    std::map<std::string, Commitment> kate_g1_elements;
-    std::map<std::string, FF> kate_fr_elements;
-    std::shared_ptr<PCSVerificationKey> kate_verification_key;
+    std::map<std::string, Commitment> pcs_g1_elements;
```

Comment:
I think the name is a bit misleading, since commitments don't have to be g1 elements for generic commitment schemes